.editorconfig

```diff
@@ -1,13 +1,17 @@
 # Stop .editorconfig files search on current file
 root = true
 
-# Unix-style newlines with a newline ending every file
 [*]
+indent_style = space
+indent_size = 2
 end_of_line = lf
-insert_final_newline = true
+charset = utf-8
 trim_trailing_whitespace = true
+insert_final_newline = true
+max_line_length = 2000
```

Comment:
This key doesn't seem standard. Do editors respect this? Also the value of `2000` looks a bit random to me.

Reply:
I do not use an IDE.
`eclint` in my CI workflow respects it.

Reply:
This feature request has 56 👍🏻-s.
https://github.com/editorconfig/editorconfig-vscode/issues/272